frontend/src/scenes/insights/BreakdownFilter.js

```diff
@@ -147,7 +147,7 @@ export function BreakdownFilter({ filters, onChange }) {
                 <Button
                     shape="round"
                     type={breakdown ? 'primary' : 'default'}
-                    disabled={shown_as == 'Stickiness'}
+                    disabled={shown_as == 'Stickiness' || shown_as == 'Lifecycle'}
```

Comment:
I can still click it despite being disabled

Reply:
fixed!